science/netcdf-fortran/Portfile

```diff
@@ -4,15 +4,14 @@ PortSystem                  1.0
 PortGroup                   muniversal 1.0
 PortGroup                   mpi 1.0
 PortGroup                   github 1.0
-# Although CMakeLists.txt is provided nf-config is not yet ported.
```

Comment:
Any reason to remove the comment? To me it looks as if the author planned migration to CMake which was not yet possible.

Reply:
Ah, now I understand to comment. I did not understand it. I rephrased it to make clear that is the reason for using configure/make instead of cmake.

Reply:
Regarding the dependents: I did check that. Of course, the dependency on the `netcdf` port brings its own dependencies.